### PR TITLE
Show runtime on version command

### PR DESCRIPTION
**Pull request by SpacePossum**
Show the PHP runtime on version command (php-cs-fixer -V), simplify code.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -10,8 +10,7 @@ labels: kind/bug
   Before reporting a bug, make sure PHP CS Fixer is up-to-date.
 
   Please describe the problem and provide technical details such as:
-   * PHP version: php -v
-   * PHP CS Fixer version: php-cs-fixer -V
+   * PHP CS Fixer version and PHP runtime: php-cs-fixer -V
```
Comment (keradus) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]: ❤️

File: src/Console/Application.php [hunk 2]
```diff
@@ -122,7 +122,8 @@ public function getLongVersion(): string
         $version = implode('', [
             parent::getLongVersion(),
             self::VERSION_CODENAME ? sprintf(' <info>%s</info>', self::VERSION_CODENAME) : '', // @phpstan-ignore-line to avoid `Ternary operator condition is always true|false.`
-            ' by <comment>Fabien Potencier</comment> and <comment>Dariusz Ruminski</comment>',
+            ' by <comment>Fabien Potencier</comment> and <comment>Dariusz Ruminski</comment>.',
+            ' PHP runtime: <comment>'.PHP_VERSION.'</comment>',
```
Comment (keradus) [inline src/Console/Application.php:126]: can we keep the new line? I would love to add extra info over time, and having everything in one liner would make it super long

```suggestion
            "\n".' PHP runtime: <comment>'.PHP_VERSION.'</comment>',
```

Comment (keradus) [inline src/Console/Application.php:126]:
```suggestion
            ' PHP runtime: <info>'.PHP_VERSION.'</info>',
```

to remain existing colour schema

Comment (SpacePossum) [reply src/Console/Application.php]: thanks for the review, I've added a commit with more changes, please have a look :)

**Review by keradus — APPROVED** [review]
looking good, but this is one of very few PRs that faces the failure on smoke tests. please take a look ;)

**Pull request closed without merging**